### ISSUE TITLE
Windows Defender positive: Redirector.GPAY!MTB

### ISSUE THREAD

**Issue by funiel** [zen-browser/desktop#3402]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Upon starting Zen it tests positive for `Trojan:HTML/Redirector.GPAY!MTB` in Windows Defender. There are no Extensions or Zen Mods active. This happens every time the browser is started.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.22

### What platform are you seeing the problem on?

Windows

### Relevant log output

```shell
Affected items:
containerfile: C:\Users\USERNAME\AppData\Local\zen\Profiles\b2jeerjy.Default (alpha)\cache2\entries\5ABEB894BA0C01D708DD715E3D5D458F23540231
file: C:\Users\USERNAME\AppData\Local\zen\Profiles\b2jeerjy.Default (alpha)\cache2\entries\5ABEB894BA0C01D708DD715E3D5D458F23540231->(SCRIPT0000)
```

**Comment by JSGRD22** [comment]
Duplicate of #37.

**Comment by syscat29** [comment]
TLDR; Windows often gives false positives, keep track in the issue mentioned above.
